python/demos/demo_stokes_nest.py

```diff
@@ -354,7 +360,7 @@ def T(u: Expr, p: Expr, mu: Expr):
         d = scipy.sparse.linalg.spsolve(KTAK, reduced_L)
         # Back substitution to full solution vector
         uh_numpy = K @ d
-        assert np.allclose(np.linalg.norm(uh_numpy, 2), np.linalg.norm(up_mpc, 2))
+        np.testing.assert_allclose(np.linalg.norm(uh_numpy, 2), np.linalg.norm(up_mpc, 2), atol=tol, rtol=10)
```

Comment:
`rtol` here is suspiciously large. Is there something I don't understand about its use?

Reply:
Uups, meant tol here